aria_extension_tests/cloudify_aria_adapters/test_context_adapter.py

```diff
@@ -304,6 +303,13 @@ def mock_workflow(ctx, graph):
                     operation_name,
                     operation_kwargs=op_dict
                 )
+                if inputs:
```

Comment:
If the operation model doesn't have the inputs declared then it will raise an exception about that. Normally you would not work like this, but for the purpose of these tests we're making these operations on-the-fly and sending in inputs.